guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -27,59 +27,61 @@ endif::[]
 Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
 endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register your host through.
 ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
+. In the *Activation Keys* field, enter one or more activation keys to assign to your host.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
-Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+. If you want to use `wget` to register your host to {Project}, select the *Use wget* option.
+By default, {Project} generates a `curl` command.
+. If your host does not trust the SSL certificate of your {ProjectServer}, select the *Insecure* option.
+During the first call, your host downloads the CA file from {Project}.
+Your host will use this CA file to connect to {ProjectServer} with all future calls.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
+If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
 +
-Instead, you can manually copy and install the CA file on each host before registering the host.
+Instead, you can manually copy and install the CA file on each host before registration.
 +
 To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
 +
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on your host and enter the following commands:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # update-ca-trust enable
 # update-ca-trust
 ----
 +
-Then register the hosts with a secure `curl` command, such as:
+Then register your host with a secure `curl` command, such as:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # curl -sS https://{foreman-example-com}/register ...
 ----
 +
```

Comment:
What I'm still thinking about is if we should leave out also the secure code examples. This would also apply for the `wget` example below. What do you think @Lennonka and @maximiliankolb?
```suggestion
```

Reply:
I'd remove them.